src/components/checkoutlogin.jsx

```diff
@@ -0,0 +1,106 @@
+import React, {Component} from 'react';
+import {Form, Button} from 'react-bootstrap';
+
+//Regex to validate the email address.
+const emailRegex = RegExp(
+    /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/
+);
+
+//Validate for form errors.
+const formValid = ({ formErrors, ...rest }) => {
+    let valid = true;
```

Comment:
❕try to use names following a standard for boolean names (starting with `is`, `has`, `does`)